Source/WebCore/page/Quirks.cpp

```diff
@@ -1687,6 +1687,9 @@ bool Quirks::shouldIgnoreContentObservationForClick(const Node& targetNode) cons
         return false;
 
     RefPtr target = dynamicDowncast<Element>(targetNode);
+    if (target->getIdAttribute().startsWith("swatchColorPicker"_s))
```

Comment:
this will be a crash if target is null

Reply:
> Additionally, I think the way this is written we'd also check for `swatchColorPicker` on walmart.com?

Yes see my comment on https://github.com/WebKit/WebKit/pull/46357#discussion_r2127813579